modules/components/src/utils/saveSet.js

```diff
@@ -3,7 +3,7 @@ import { graphql } from './api';
 export default ({ type, path, userId, sqon = {}, returnIds = false, api }) =>
```

Comment:
what about adding `sort` to this function / query, which defaults to an empty array, and in the resolver below checks for both `sort` and `sort.length` before defaulting to `['_id']` (or another method of defaulting to _id). Just might be nice to keep this helper function inline with the resolver.